README-lede_generic.md

```diff
@@ -15,3 +15,32 @@ The agent has its own version that is generated if it is built separately
 from the firmware (only happens during development, it is based on time).
 For the release builds the agent version and the firmware version are the
 same.
+
+## Platforms
+
+Following platforms are supported so far.
+ - TPLink Archer C7 v2/v4/v5
+ - GL - Inet B1300
+ - GL - Inet AR300M
```

Comment:
We aren't building AR300M and Linksys.
You can list them as "support being phased out" along with Smatrg SR400 (there are still files for all 3).

Reply:
OK. Done.